### PR TITLE
Update cross-media-measurement-api version.

### PR DESCRIPTION
**Pull request by efoxepstein**
This also moves the internal ExchangeStepAttempts service to be in
parity with the public API version.

---
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/51)

### CODE REVIEW

**Review by stevenwarejones — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/51)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @efoxepstein and @yunyeng)

___
*[src/main/kotlin/org/wfanet/measurement/kingdom/service/api/v2alpha/ExchangeStepsService.kt, line 30 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/51#-M_1KbgF3ofzwD_0_qun:-M_1KbgF3ofzwD_0_quo:bgdwvry) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/e2eaada8990685b04221d3761bb2697402988abd/src/main/kotlin/org/wfanet/measurement/kingdom/service/api/v2alpha/ExchangeStepsService.kt#L30)):*
> ```Kotlin
>   }
> 
>   override suspend fun getExchangeStep(request: GetExchangeStepRequest): ExchangeStep {
> ```

what is the difference between claimReady and Get?

**Review by efoxepstein — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/51)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @stevenwarejones and @yunyeng)

___
*[src/main/kotlin/org/wfanet/measurement/kingdom/service/api/v2alpha/ExchangeStepsService.kt, line 30 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/51#-M_1KbgF3ofzwD_0_qun:-M_1Krk896FsN9VFQ9bc:b-fg660u) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/e2eaada8990685b04221d3761bb2697402988abd/src/main/kotlin/org/wfanet/measurement/kingdom/service/api/v2alpha/ExchangeStepsService.kt#L30)):*
<details><summary><i>Previously, stevenwarejones (Steven Ware Jones) wrote…</i></summary><blockquote>

what is the difference between claimReady and Get?
</blockquote></details>

`get` is a standard API method (https://google.aip.dev/131) that returns a specific resource given its id.

In other words, it translates `ExchangeStep.Key` into `ExchangeStep`. It has no side effects.

`claimReady` returns an "arbitrary" `ExchangeStep` that belongs to the party in the request that's in state `READY` or `READY_FOR_RETRY`.

**Review by stevenwarejones — APPROVED** [review]
> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/51)* status: all files reviewed, 1 unresolved discussion (waiting on @stevenwarejones and @yunyeng)
